python/hopsworks/command.py

```diff
@@ -20,19 +20,25 @@
 class Command:
     def __init__(
         self,
+        id=None,
         status=None,
         op=None,
         install_type=None,
         error_message=None,
+        custom_commands_file=None,
+        args=None,
         type=None,
         href=None,
         count=None,
     ):
+        self._id = id
         self._op = op
         self._install_type = install_type
         self._status = status
         self._error_message = error_message
         self._count = count
+        self._custom_commands_file = custom_commands_file
```

Comment:
Would be nice to leave a comment why they should be added so that we don't regress.